Doc/library/sqlite3.rst

```diff
@@ -1975,6 +1976,16 @@ both styles:
 
    [('C', 1972)]
 
+.. note::
+
+   :pep:`249` numeric placeholders are not supported.
+   If used, they will be interpreted as named placeholders.
+
+Although **strongly discouraged**, for the named style,
+``parameters`` can be supplied as a :term:`!sequence`.
+This will result in the parameters being blindly bound to the placeholders
+simply by index; parameter names will silently be ignored.
```

Comment:
Perhaps this is too much for a how-to. We could add a brief explanation and throw it in there together with more details on how sqlite3 chooses between the two supported styles.

Reply:
```suggestion
```